CODE_OF_CONDUCT.md

```diff
@@ -18,47 +48,63 @@
 * Promote and teach respectful, effective communication.
 * Welcome contributions to the project in all the forms they take.
 
-## Prohibited Behavior 
+## Prohibited Behavior
 
-* No discrimination or comments promoting or reinforcing the oppression of any groups or people based on gender, gender expression, race, ethnicity, nationality, sexuality, religion, disability, mental illness, neurodivergence, personal appearance, physical appearance, body size, age, or class.
-* No conduct intended to stalk, harass, or intimidate other users.
-* Do not continue to engage in conversation with a user that has specifically requested for said engagement to cease regardless of platform-specific privacy tools employed.
-* No inciting others to engage another user in continued interaction or discussion after a user has requested for said engagement with that user to cease.
-* No xenophobia or violent nationalism.
+There is also going to be behaviour that will be more strictly prohibited.
+
+### Discrimination
+
+* Do not participate in discrimination or comments promoting or reinforcing the existing systems of oppression of any groups or people based on gender, gender expression, race, ethnicity, nationality, sexuality, religion, disability, mental illness, neurodivergence, personal appearance, physical appearance, body size, age, or class.
 * Do not claim “reverse-isms,” e.g. “reverse racism”
-* No sexual or violent imagery without a content warning.
-* No sexualized depictions of minors, in any way. (This includes drawings and 3D renders).
-* No aggregating, posting, and/or disseminating a person’s demographic, personal, or private data without express permission.
-* No posting and/or disseminating a person’s posts, including screen captures or any other content without express permission.
-* No posting or disseminating libel, slander, or other disinformation.
+* Do not participate in xenophobia or violent nationalism.
+
+### Harassment and privacy violation
+
+* Do not contribute to behaviour intended to stalk, harass, or intimidate other users.
+* Do not continue to engage with a user that has specifically asked you to stop, whether or not they have blocked or muted you. Do not ask others to engage with them on your behalf.
```

Comment:
"regardless of whether" might be better than "whether or not"

Reply:
would this be fine to be a minor edit later?

Reply:
Indeed. These comments should be taken in the spirit of "are optional things you can change if you want to". Particularly if this is just a draft rather than the final thing :)